tests/integration/clientv3/naming/resolver_test.go

```diff
@@ -65,10 +65,12 @@ func TestEtcdGrpcResolver(t *testing.T) {
 		t.Fatal("failed to add foo", err)
 	}
 
-	r := &etcdnaming.GRPCResolver{Client: clus.Client(1)}
-	b := grpc.RoundRobin(r)
+	b, err := resolver.NewBuilder(clus.Client(1))
+	if err != nil {
+		t.Fatal("failed to new resolver builder", err)
+	}
 
-	conn, err := grpc.Dial("foo", grpc.WithInsecure(), grpc.WithBalancer(b))
+	conn, err := grpc.Dial("etcd:///foo", grpc.WithInsecure(), grpc.WithResolvers(b))
```

Comment:
I'm curious ? Do we need tripple: etcd:///foo or 'etcd://foo' is sufficient

Please update: 
https://github.com/etcd-io/website/blob/master/content/docs/current/dev-guide/grpc_naming.md if we need '///'

Reply:
Yes. According to this official doc: https://github.com/grpc/grpc/blob/master/doc/naming.md#name-syntax, gRPC cannot parse `etcd://foo` properly to get the expected scheme `etcd`.